src/core/tracking.js

```diff
@@ -33,6 +33,10 @@ import {
 import log from 'core/logger';
 import { convertBoolean } from 'core/utils';
 
+export const PROMOTED_ADDON_EVENT_URL = `${config.get('apiPath')}${config.get(
```

Comment:
I don't understand why this value is in this file.

Reply:
It's going to be needed in two components, including `installAddon`, so it needs to be somewhere in `core` and not `amo`, which is why I didn't just put it in the component. But... I can change the way I'm using it in `installAddon` so that it's not needed there. It's in `tracking` because its only purpose is for use with `sendBeacon`, which is also in `tracking`.

I think what I need to do to resolve this is to create another function in `tracking` whose purpose is to send this type of beacon, and it will make use of this URL. Then `installAddon` can just call that without having to worry about any URLs at all.

Reply:
The `tracking` code and its `sendBeacon` should not be tied to a particular feature so I don't expect to have a constant like the one above in this file. `sendBeacon` might be used for other purposes in the feature.

This patch is unrelated to this constant so I'd rather move it back to where it was and once we have a need to share it, we can think where to put it.

> I think what I need to do to resolve this is to create another function in tracking whose purpose is to send this type of beacon, and it will make use of this URL. Then installAddon can just call that without having to worry about any URLs at all.

Yes, a specialized function would work too, I guess. You might also want to use the hardcoded value instead of `config.get()`.

Reply:
I created a specialized function which can then also be used by `installAddon`.